recipe/meta.yaml

```
@@ -23,7 +23,7 @@ requirements:
     - python
     - dask
     - h5py
-    - tifffile
+    - tifffile >=2020.08.25
```

The package's versioning scheme is not zero-padded, unfortunately. See https://github.com/nsls-ii-forge/tifffile-feedstock/blob/73194f5a61730a46c82ef3f356625d94ab1cae62/recipe/meta.yaml#L2.
```suggestion
    - tifffile >=2020.8.25
```